From ec79f208f922b077250d8e52f1908542288d90fa Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Mon, 26 Jun 2017 21:18:00 +0200
Subject: [PATCH] fix SDK paths (especially cat/sed) in bin/distro-install-sdk

wonder how that ever worked. cat and sed are in /bin, not /usr/bin.
Appparetly the BSDs have them there, though, so generalize though this
is for distros and it probably shouldn't matter.

Change-Id: I6efb3e92b8f8d15f3671090a3eaa9d8836f9998f
---

diff --git a/bin/distro-install-sdk b/bin/distro-install-sdk
index a92c3b5..e8cf28d 100755
--- a/bin/distro-install-sdk
+++ b/bin/distro-install-sdk
@@ -68,11 +68,11 @@
     sed -e "s,@OO_SDK_NAME@,libreoffice${PRODUCTVERSION}_sdk," \
         -e "s,@OO_SDK_HOME@,$INSTALLDIR/sdk," \
         -e "s,@OFFICE_HOME@,$INSTALLDIR," \
-        -e "s,@OO_SDK_MAKE_HOME@,/usr/bin," \
-        -e "s,@OO_SDK_ZIP_HOME@,/usr/bin," \
-        -e "s,@OO_SDK_CPP_HOME@,/usr/bin," \
-        -e "s,@OO_SDK_SED_HOME@,/usr/bin," \
-        -e "s,@OO_SDK_CAT_HOME@,/usr/bin," \
+        -e "s,@OO_SDK_MAKE_HOME@,$(dirname $(command -v make))," \
+        -e "s,@OO_SDK_ZIP_HOME@,$(dirname $(command -v zip))," \
+        -e "s,@OO_SDK_CPP_HOME@,$(dirname $(command -v cpp))," \
+        -e "s,@OO_SDK_SED_HOME@,$(dirname $(command -v sed))," \
+        -e "s,@OO_SDK_CAT_HOME@,$(dirname $(command -v cat))," \
         -e "s,@OO_SDK_JAVA_HOME@,$JAVA_HOME," \
         -e "s,@OO_SDK_OUTPUT_DIR@,\$HOME," \
         -e "s,@SDK_AUTO_DEPLOYMENT@,NO," \
